eth/downloader/peer.go

```diff
@@ -585,13 +589,26 @@ func (ps *peerSet) RTTs() []time.Duration {
 		rtts = append(rtts, p.rtt)
 		p.lock.RUnlock()
 	}
-	// Sort into ascending order and return them
+	// Sort into ascending order and retrieve the median
 	for i := 0; i < len(rtts); i++ {
 		for j := i + 1; j < len(rtts); j++ {
 			if rtts[i] > rtts[j] {
 				rtts[i], rtts[j] = rtts[j], rtts[i]
 			}
 		}
 	}
```

Comment:
Switched the `rtts` array to float64 and used `sort`, will push in the next commit. I'd rather keep `peer.rtt` as a time as I don't think a single conversion once every few seconds matters much and it's less error prone and more descriptive to have a meaningful `time.Duration` type for an RTT value.